packages/navigation_legacy/package.json

```diff
@@ -37,12 +37,10 @@
         "@tonlabs/uikit.components": "^2.3.0",
         "@tonlabs/uikit.core": "^2.3.0",
         "@tonlabs/uikit.hydrogen": "^2.3.0",
-        "prop-types": "15.6.2",
         "react-image-lightbox": "5.1.1",
```

Comment:
Need not to forget removing it ;)

Reply:
What do you mean? :)

Reply:
Just kill it from the project =)

Reply:
Oh, I thought you commented on `prop-types` :D